Makefile.am

```diff
@@ -2,9 +2,9 @@ ACLOCAL_AMFLAGS=-I m4
 
 SUBDIRS = src tests
 
-if WITH_LIBCTL
-  SUBDIRS += scheme
-endif
+#if WITH_LIBCTL
+#  SUBDIRS += scheme
+#endif
```

Comment:
I assume this was commented out for testing purposes and was meant to be reverted?

Reply:
fixed